src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -434,7 +434,6 @@ static bool _appendClipShape(SvgLoaderData& loaderData, SvgNode* node, Shape* sh
         }
     }
 
-    _applyProperty(loaderData, node, shape, vBox, svgPath, true);
```

Comment:
This even properly handles the return value that could potentially a memory leak.